pkg/sql/planner.go

```diff
@@ -235,6 +235,12 @@ func makeInternalPlanner(
 }
 
 func finishInternalPlanner(p *planner) {
+	// If we're panicking, don't bother stopping the MemoryMonitors. We're likely
+	// in a state where not all memory has been properly release to the monitors,
+	// so Stop() would panic again and add confusion to stack traces.
```

Comment:
it's generally not a good idea to refer to specific bindings or names like `MemoryMonitors` and `Stop`; if these names change, the comment will have rotted, and the names' presence does not add much clarity in a majority of cases.